### PR TITLE
Capistrano Deploy Configuration [v2]

### PR DESCRIPTION
**Pull request by akaplo**
Adds Capistrano for deploying to the web a-la #303.

@werebus I'll need you to review.  I'm missing steps (i.e. the `npm run ionic:build --prod --release`) and I possibly have some directories wrong (see my comments).

~~This isn't ready to be merged (is _mostly_ ready), but I wanted a PR so we could have a centralized place for talking about this (we both get off-topic too easily in person, which makes for some simple conversations that take over a week).~~

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+set :application, 'pvta-multiplatform'
+set :scm, :scm
+set :branch, 'master'
+set :repo_url, 'git@github.com:umts/pvta-multiplatform.git'
+
+set :local_temp_dir, Pathname.new(Dir.mktmpdir)
+set :deploy_to, "/srv/#{fetch :application}"
+
+set :app_subdirectory, 'www'
```
Comment (akaplo) [inline config/deploy.rb]: Should this be `src` now?

File: lib/capistrano/tasks/npm.rake [hunk 2]
```diff
@@ -0,0 +1,9 @@
+namespace :npm do
+  task :install do
+    run_locally do
+      within fetch(:local_temp_dir).join('release') do
+        execute :npm, 'install'
+      end
```
Comment (akaplo) [inline lib/capistrano/tasks/npm.rake:6]: Should we invoke the build command here?

File: config/deploy.rb (hunk 1 above)
Comment (werebus) [reply config/deploy.rb]: No

**Review by werebus — APPROVED** [review]
Seems to work for me

Although, I'm maybe not the best reviewer since I wrote much of it.

File: lib/capistrano/tasks/npm.rake (hunk 2 above)
Comment (akaplo) [reply lib/capistrano/tasks/npm.rake:6]: All good, see below.

File: src/app/ga.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+export var gaNewElem: any = {};
```
Comment (akaplo) [inline src/app/ga.ts:1]: Did you end up trying this without these empty variables?

**Review by akaplo — COMMENTED** [review]
Looks great to me.  I have one question that I left in a comment, but its answer doesn't affect whether I approve (which `I do!`...I can't "officially" approve, since I opened the PR).

File: src/app/ga.ts (hunk 3 above)
Comment (werebus) [reply src/app/ga.ts:1]: Nah. I think it's fine.

**Comment by werebus** [comment]
Well, here goes then.

**Pull request merged**